# -*- coding: utf-8 -*-
# Generated by Django 1.11.3 on 2017-08-11 09:46
from __future__ import unicode_literals

import datetime
import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('usermanage', '0001_initial'),
    ]

    operations = [
        migrations.AlterField(
            model_name='clients',
            name='created_time',
            field=models.DateTimeField(default=datetime.datetime(2017, 8, 11, 15, 16, 33, 562767)),
        ),
        migrations.AlterField(
            model_name='clients',
            name='modified_time',
            field=models.DateTimeField(default=datetime.datetime(2017, 8, 11, 15, 16, 33, 562819)),
        ),
        migrations.AlterField(
            model_name='countries',
            name='created_time',
            field=models.DateTimeField(default=datetime.datetime(2017, 8, 11, 15, 16, 33, 559929)),
        ),
        migrations.AlterField(
            model_name='countries',
            name='modified_time',
            field=models.DateTimeField(default=datetime.datetime(2017, 8, 11, 15, 16, 33, 559988)),
        ),
        migrations.AlterField(
            model_name='users',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.core.validators.RegexValidator('^[\\w.@+-]+$', 'Enter a valid username. This value may contain only letters, numbers and @/./+/-/_ characters.', 'invalid')], verbose_name='usernAme'),
        ),
    ]
